CMakeLists.txt

```diff
@@ -336,10 +336,17 @@ else()
     endif()
 
     if(QUIC_ENABLE_LOGGING)
-        message(STATUS "Configuring for LTTng tracing")
-        set(CMAKE_CLOG_CONFIG_PROFILE linux)
-        list(APPEND QUIC_COMMON_DEFINES QUIC_CLOG)
-        include(FindLTTngUST)
+        if (APPLE)
+            message(STATUS "Configuring for macos tracing")
+            set(CMAKE_CLOG_CONFIG_PROFILE macos)
+            #set(CMAKE_CLOG_CONFIG_PROFILE stubs)
```

Comment:
Might be better to default to `stubs` instead of `macos` if we are going to use STDOUT for `macos`.